bob/src/grinder.rs

```diff
@@ -199,6 +199,26 @@ impl Grinder {
         self.counter.spawn_task(self.backend.clone());
         self.hw_counter.spawn_task();
     }
+
+    pub(crate) async fn delete(&self, key: BobKey, with_aliens: bool) -> Result<u64, Error> {
+        trace!(">>>- - - - - GRINDER DELETE START - - - - -");
```

Comment:
It would be good to add `GRINDER_DELETE_COUNTER` to track number of deleted records. Also it would be good to add `GRINDER_DELETE_TIMER` to track delete time in Graphite